src/hooks/usePickerContext.ts

```diff
@@ -1,19 +1,25 @@
 import { Dispatch, SetStateAction, useState } from 'react';
 
 export interface PickerContextType {
-    searchTerm: string;
-	showTrending: boolean
+	searchTerm: string;
+	showTrending: boolean;
+	initialSearchTerm: string;
```

Comment:
I'm using VS code default format, it was odd to the previous code, seems okay now

Reply:
Thanks, i will work on this kind of issues https://github.com/MrBartusek/gif-picker-react/issues/34